components/match2/wikis/starcraft2/legacy/match_maps_team_legacy.lua

```diff
@@ -107,19 +107,24 @@ function MatchMapsTeamLegacy._processMapOpponent(side, prefix, mapArgs, archon)
 		['t' .. side .. 'p3flag'] = _match2Args[prefix .. 't' .. side .. 'p3flag'],
 	}
 
-	MatchMapsTeamLegacy._setPlayersForOpponents(addToMapArgs, side, _match2Args[prefix .. 'p' .. side])
+	MatchMapsTeamLegacy._setPlayersForOpponents(addToMapArgs, side, {
+			_match2Args[prefix .. 'p' .. side],
```

Comment:
This can never been entered as `prefix .. 't' .. side .. 'p1'`?

Reply:
no the old match maps team did not support that so can not happen